packages/plugin-board/test/integration/spec/board.js

```diff
@@ -0,0 +1,414 @@
+  this.timeout(30000);
+
+  let board, conversation, fixture, participants;
+  const mercuryBindingsPrefix = `board.`;
+
+  // create users
+  before(() => testUsers.create({count: 3})
+    .then((users) => {
+      participants = users;
+
+      return Promise.all(map(participants, (participant) => {
+        participant.spark = new CiscoSpark({
+          credentials: {
+            authorization: participant.token
+          }
+        });
+        return participant.spark.device.register();
+      }));
+    })
+
+    // create conversation
+    .then(() => participants[0].spark.conversation.create({
+      displayName: `Test Board Conversation`,
+      participants
+    }))
+    .then((c) => {
+      conversation = c;
+      return conversation;
+    })
+
+    // create channel (board)
+    .then(() => participants[0].spark.board.createChannel({aclUrl: conversation.id}))
+    .then((channel) => {
+      board = channel;
+      return channel;
+    })
+
+    // connect to realtime channel
+    .then(() => {
+      const mercuryBindingId = boardChannelToMercuryBinding(board.channelId);
+      const bindingStr = [mercuryBindingsPrefix + mercuryBindingId];
+      const bindingObj = {bindings: bindingStr};
+
+      return Promise.all(map(participants, (participant) => {
+        return participant.spark.board.register(bindingObj)
+          .then((url) => {
+            participant.spark.board.realtime.set({
+              boardWebSocketUrl: url.webSocketUrl,
+              boardBindings: bindingStr
+            });
+            return participant.spark.board.realtime.connect();
+          });
+      }));
+    })
+
+    // load fixture image
+    .then(() => fh.fetch(`sample-image-small-one.png`))
+
+    .then((fetchedFixture) => {
+      fixture = fetchedFixture;
+      return fetchedFixture;
+    })
+
+    // confirm we are listening
+    .then(() => Promise.all(map(participants, (participant) => {
+      assert(participant.spark.board.realtime.connected, `participant is not listening`);
+    }))));
+
+  // disconnect realtime
+  after(() => Promise.all(map(participants, (participant) => {
+
+    if (participant.spark.board.realtime.connected) {
+      return participant.spark.board.realtime.disconnect()
+        .then(() => {
+          participant.spark.board.realtime.set({boardWebSocketUrl: ``});
+          participant.spark.board.realtime.set({boardBindings: []});
+        });
+    }
+    return true;
+  })));
+
+  describe(`#_uploadImage()`, () => {
+
+    it(`uploads image to spark files`, () => {
+      return participants[0].spark.board._uploadImage(conversation, fixture)
+        .then((scr) => {
+          return participants[0].spark.encryption.download(scr);
+        })
+        .then((downloadedFile) => {
+          assert(fh.isMatchingFile(downloadedFile, fixture));
+        });
+    });
+  });
+
+
+  describe(`#ping()`, () => {
+
+    it(`pings board service`, () => {
+      return participants[0].spark.board.ping()
```

Comment:
please omit the outer braces and the return statement for test that return on line 1.